frontend/src/components/intake/ShasIntakeForm.vue

```diff
@@ -900,9 +955,23 @@ onBeforeMount(async () => {
               :click-callback="clickCallback"
               title="Permits & Reports"
               icon="fa-file"
+              :class="{
+                'p-red':
```

Comment:
The `p-` prefix is a primevue directive. We should not be using it in our own code. `error-red` or something would be a better fit.